app/src/main/java/org/mozilla/focus/browser/cursor/CursorViewModel.kt

```diff
@@ -62,7 +62,7 @@ class CursorViewModel(
                 isInitialPosSet = true
                 pos.set(value.x / 2, value.y / 2) // Center.
             }
-            onUpdate(pos.x, pos.y, getScrollVel())
+            onUpdate(pos.x, pos.y, getPercentMaxScrollVel(), 1f)
```

Comment:
Why is `onUpdate called from within `onUpdate`? or am I reading this wrong.

Reply:
`onUpdate` is called from the setter of `maxBounds`, not `onUpdate`.